README.md

```diff
@@ -16,3 +16,57 @@ Also give a theoretical argument for why X could or could not be correct, based
 on the complexity of the general sorting problem we covered in class.
 
 Add your answers to this markdown file.
+
+## Answer 
+Testing: 
+- To determine if the algorithm operates in linear time.
+- I would use a set of test cases to check if the algorithm's runtime scales proportionally with an increasing input size.
+- These inputs would be arrays of various input types.
+- Some of the case types I would use would be an already sorted array, reverse sorted array, randomly generated array, array with 
+  duplicate values, and an empty array.
+- Each of these input types would be tested with different sizes to observe the runtime scaling as the input sizes increase.
+- This would allow us to see if the runtime acts in a manner greater than $O(n)$.
+- I would expect that the initial time complexity is incorrect.
```

Comment:
Not sure what you mean by this -- what's the "initial time complexity"?

Reply:
What I meant by initial time complexity is the  given time complexity of O(n). I simple removed that bullet as it seemed repetitive.